spec/defines/foreman_rake_spec.rb

```diff
@@ -5,7 +5,7 @@
 
   context 'on RedHat' do
     let :facts do
-      on_supported_os['redhat-8-x86_64']
+      on_supported_os['redhat-9-x86_64']
```

Comment:
I wonder if we could reuse `DEFAULT_OS_FACTS` from `spec/support/plugin.rb`.

Reply:
Does it also make sense to limit the facts here at all, or just run them on all OSes in metadata.json? That will create some overhead, but I'm not sure if DEFAULT_OS_FACTS will always be updated properly.

Reply:
I think we could even do something like `on_supported_os.first` to just take the first one. Then it comes down to something like:
```ruby
os, os_facts = on_supported_os.first
context "on #{os}" do
  let(:facts) { os_facts }
```

Reply:
updated it